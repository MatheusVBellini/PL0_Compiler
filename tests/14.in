CONST abc=1;
VAR 1, a;
.