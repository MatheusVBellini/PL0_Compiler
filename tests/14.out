2:5 error: Invalid identifier

Compilation failed, 1 errors
